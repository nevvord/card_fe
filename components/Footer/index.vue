<template lang="pug">
  footer
    .container
      p
      | &copy; 2020 | Шо картишки
</template>
<style lang="scss" scoped>
@import '@/assets/scss/_variables.scss';

footer {
  background-color: map-get($mainColors, black );
  text-align: center;
  color: #fff;
  padding: 1rem;
}
</style>
