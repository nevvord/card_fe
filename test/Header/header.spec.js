import { mount } from '@vue/test-utils'
import Footer from '@/components/Footer'

describe('Footer', () => {
  const wrapper = mount(Footer)
  it('Render the correct marup', () => {
    expect(wrapper.html()).toContain('<div class="test">0</div>')
  })
  it('Has a button', () => {
    expect(wrapper.contains('button')).toBe(true)
  })
  it('Button click should increment the count', () => {
    expect(wrapper.vm.test).toBe(0)
    const button = wrapper.find('button')
    button.trigger('click')
    expect(wrapper.vm.test).toBe(1)
  })
  it('Button click should increment the count text', async () => {
    expect(wrapper.text()).toContain('1')
    const button = wrapper.find('button')
    await button.trigger('click')
    expect(wrapper.text()).toContain('2')
  })
})
