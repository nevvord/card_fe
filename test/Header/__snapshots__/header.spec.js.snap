// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Vue or not 1`] = `true`;
